### ISSUE TITLE
Multi-site: Parent does not exist in structure error

### ISSUE THREAD

**Issue by mikemartin** [statamic/cms#2176]
## Bug Description

The parent field seems to interfere with localising a page. Is this the expected flow or is there a better way to handle this error.
 
![2020-08-05 10 43 24](https://user-images.githubusercontent.com/414211/89359404-1bd76580-d709-11ea-9ed7-3b8a8558fd59.gif)

## Environment

**Statamic version:** 3.0.0.-beta-{?}

**PHP version:** 7.{?}

**Install method (choose one):**
- Fresh install from `statamic/statamic`
- Starter kit
- Existing Laravel app
- Other (please explain this magic!)

**Comment by sandervanh** [comment]
Same as issue #2172

**Comment by sandervanh** [comment]
@mikemartin How did you make the parent writable in localized entry? i've tried 
```
-
        handle: parent
        field:
          type: select
          localizable: true
```
But then I'm losing the right sidebar (default) functionality.

Thanks!

**Comment by mikemartin** [comment]
Hey @sandervanh!

This was a fresh install and all I needed to do was add localizable to the parent field. It looks like I'm using the `entries` field instead of the `select`. Here's the fieldset:

```
title: Pages
sections:
  main:
    display: Main
    fields:
      -
        handle: title
        field:
          type: text
          required: true
          localizable: true
      -
        handle: content
        field:
          type: markdown
          display: Content
          localizable: true
      -
        handle: author
        field:
          type: users
          display: Author
          default: current
          localizable: true
          max_items: 1
      -
        handle: template
        field:
          type: template
          display: Template
          localizable: true
  sidebar:
    display: Sidebar
    fields:
      -
        handle: slug
        field:
          type: slug
          required: true
          localizable: true
      -
        handle: parent
        field:
          type: entries
          collections:
            - pages
          max_items: 1
          listable: false
          localizable: true
  SEO:
    display: SEO
    fields:
      -
        handle: seo
        field:
          type: seo_pro
          listable: false
```

**Comment by sandervanh** [comment]
Thanks @mikemartin !

**Comment by AndreasSchantl** [comment]
There's another issue related to that one. Every time the page is updated, it's out of sync..

**Comment by jasonvarga** [comment]
Looks like this might have been solved by 20592067 which will be in 3.0.2
